tests/src/main/java/com/hazelcast/simulator/tests/queue/TxnQueueWithLockTest.java

```diff
@@ -83,17 +83,16 @@ public void run() {
                         secondLock.unlock();
 
                         ctx.commitTransaction();
+                        queue.take();
```

Comment:
This take could be ok,  as we only take,  it ctx commit is ok,  and we put in the ctx,  so we don't block